cmd/witness/cmd/run.go

```diff
@@ -15,12 +15,13 @@
 package cmd
 
 import (
+	"context"
 	"encoding/json"
 	"fmt"
 
 	"github.com/spf13/cobra"
 	"github.com/testifysec/witness/cmd/witness/options"
-	"github.com/testifysec/witness/pkg"
+	witness "github.com/testifysec/witness/pkg"
```

Comment:
since the package at witness/pkg is named witness you shouldn't need to alias it here fyi